### ISSUE TITLE
Travis build is broken with selenium-webdriver tests.

### ISSUE THREAD

**Issue by ponelat** [swagger-api/swagger-ui#1773]
By some magic, if I build with node.js 4.2.2 the tests run. If I build with 0.10 (as travis is configured to) it fails.

Initially I thought this was related to browserify, but swagger-ui just concats its files!

**Comment by ponelat** [comment]
Looks like I've narrowed it down to swagger-client@2.1.7
Might be a simple fix (like updating the IDs)

**Comment by ponelat** [comment]
So... the fix is not so simple...
Turns out the bug is quite deep... its in the browser-shim for `Buffer` https://github.com/feross/buffer/issues/79

**Comment by ponelat** [comment]
A possible workaround is to force a downgrade of `buffer` in swagger-js to something that works

**Comment by fehguy** [comment]
Eww yes we can go to 3.4.3 in js if we can verify that it works

**Comment by ponelat** [comment]
I've tested locally 
- using nvm and node.js 0.10.40 - same as travis
- using npm link to pull in swagger-js with the fix (https://github.com/swagger-api/swagger-js/pull/637)

PS: this didn't just break the tests. It broke Swagger-UI. I tested the above by serving up swagger-ui, as well.

swagger-ui@master ( with swagger-client@2.1.7 ) , built on node.js <= 0.12.x would produce a browser error and stall.

**Comment by fehguy** [comment]
how could it break swagger-ui?  there is no node process in the runtime of that, unless it's causing it to build incorrectly.

**Comment by ponelat** [comment]
at build time, it adds the erroneous module to it.

TLDR;
I built with node 0.10 and added the files to this branch... https://github.com/swagger-api/swagger-ui/tree/showcase/built-with-0.10
Serve it and see, you'll get a browser error that stalls it.

To repeat...

> Where pwd is swagger-ui
```sh
 cd to .../../.swagger-ui
 docker run -it -v `pwd`:/usr/app node:0.10 bash
 cd /usr/app npm i && npm run build
 exit
 # Serve swagger-ui/dist and see
```

**Comment by Udayakumarg** [comment]
Hi All,


I am trying to Automate Swagger using selenium. Seleinium is not able to identify the elements, Please let me know what are the setup's required to do.

**Comment by ponelat** [comment]
I imagine you might need to wait for the javascript to render the page first. Once its loaded, you can use any old selector to find the elements.
